pkg/tenant/tenants_test.go

```diff
@@ -50,6 +50,18 @@ func (suite *tenantTestSuite) TestTenantCreate() {
 		return &pb.PutReply{}, nil
 	}
 
+	// Should return an error if tenant id exists.
```

Comment:
`requireError` was added after this test was approved, so that's why the tests are being added now.